JPS_BASE_LIB/src/test/java/uk/ac/cam/cares/jps/base/timeseries/TimeSeriesRDBClientIntegrationTest.java

```diff
@@ -32,15 +28,14 @@ public class TimeSeriesRDBClientIntegrationTest {
 	// Define RDB database setup (analogous to a triple-store endpoint)
 	// Using special testcontainers URL that will spin up a Postgres Docker container when accessed by a driver
 	// (see: https://www.testcontainers.org/modules/databases/jdbc/). Note: requires Docker to be installed!
-	private static final String dbURL = "jdbc:tc:postgresql:13.3:///timeseries";
+	private static final String dbURL = "jdbc:tc:postgresql:9.6.8:///timeseries";
```

Comment:
Is there a reason for using an older version here?

Reply:
The reason for changing the version is to make it unique and different from the ones used in the existing tests. This also solves the problem of failing tests and solves the error of a data iri having an existing timeseries instance.

Reply:
This is not a good solution. 
It looks by default the testcontainer class tries to reuse an existing container to increase performance. You should stop the container at the end of the test class, there should be more than 1 way to do this.

Reply:
There should be a method to clear all time series data, if you use that at in an @AfterClass method that should work too

Reply:
done